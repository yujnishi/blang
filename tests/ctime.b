now[2] 0,0;

main() {
    auto buf "Mmm dd hh:mm:ss";

    ctime(now,buf);
    printf("now1 is *'%s*'*n",buf);

    now[0] = 1586066828;
    ctime(now,buf);
    printf("now2 is *'%s*'*n",buf);
}
