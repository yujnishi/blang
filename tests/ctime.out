now1 is 'Jan 01 00:00:00'
now2 is 'Apr 05 06:07:08'
